utils/documentOps.py

```diff
@@ -0,0 +1,225 @@
+#!/usr/local/bin/python3
+
+# SPDX-License-Identifier: Apache-2.0
+
+##################### documentOps.py ########################################
+#
+# Copyright 2022 The IBM Research Authors.
+#
+################################################################################
+#
+# This file convert .mlir from stdin into a FileCheck file format. It also
```

Comment:
Update the header comment, which seems to be copied from mlir2FileCheck.py

Reply:
good catch, sorry.